openmc/cell.py

```diff
@@ -45,6 +46,11 @@ class Cell(IDManagerMixin):
         Indicates what the cell is filled with.
     region : openmc.Region or None
         Region of space that is assigned to the cell.
+    color : {'str', 'tuple'}
+        Color of this Cell when 'color_by == cell'.
+        Can be a named color or an RGB tuple.
```

Comment:
```suggestion
        Can be a named color or an RGB tuple containing
        three integer values which are each between 0 and 255.
```
Do we want to let people know the tuple range as sometimes RGB is from 0 to 1. I'm sure there is a more elegant way of putting this but I had a go at adding to the docstring.

Reply:
This wording makes sense to me. I've added to _cell.py_ and _material.py_ and clarified the first sentence a bit more.